### DISCUSSION TITLE
Superscript/SubScript component

### DISCUSSION THREAD

**Discussion by johnhbarrett** [Megabit/Blazorise#3888] [Ideas]
How about a component that can render subscript or superscript?

**Comment by stsrki** [comment]
I'm not sure it is best to have a component just for this one thing. And especially since it can relatively easily be done with a few lines of CSS.

https://stackoverflow.com/questions/501671/superscript-in-css-only

**Comment by johnhbarrett** [comment]
Thanks @stsrki - 
How about if I have say a Grid with a particular column I wish to use a "MarkDown" component - is that possible?

So start with this...

```

<DataGridColumn TItem="ActivityUnitWithNavigationPropertiesDto" 
                Field="ActivityUnit.Symbol"
                Caption="@L["Symbol"]">
</DataGridColumn>

```

Trying to use a 'MarkDown'  Component

```
@convertedMarkdown

@code{
    // Use MarkupString type to prevent Blazor from using HtmlEncoder.Encode
    private MarkupString convertedMarkdown;

    [Parameter]
    public string Text { get; set; }

    protected override void OnParametersSet()
    {
    // Required Markdig https://www.nuget.org/packages/Markdig/
        var html = Markdig.Markdown.ToHtml(Text ?? "");
        convertedMarkdown = (MarkupString)html; // or new MarkupString(html)
    }
}
```
inside the Grid - like this - but does not work for me...

 ```
<DataGridColumn TItem="ActivityUnitWithNavigationPropertiesDto" 
       Caption="@L["Symbol"]">
      <Field  >
           <MarkDown Text="ActivityUnit.Symbol"></MarkDown>
      </Field>
</DataGridColumn>
```

`<PackageReference Include="Markdig" Version="0.30.2" />`

**Reply by johnhbarrett** [reply]
ah - just discovered `DisplayTemplate` :-)
